### PR TITLE
Esquery migrate to 1.x features

### PR DESCRIPTION
**Pull request by snopoke**
- fields -> source
- facets -> aggregations

can do :fish: but there's a bit of :back: and :fast_forward: so :post_office: also OK

### CODE REVIEW

**Comment by dimagimon** [comment]
There are 26 errors:

* corehq/apps/analytics/tasks.py, line 275 - E501 line too long (122 > 115 characters)
* corehq/apps/es/tests/test_aggregations.py, line 60 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 225 - E122 continuation line missing indentation or outdented
* corehq/apps/es/tests/test_aggregations.py, line 237 - W391 blank line at end of file
* corehq/apps/es/tests/test_esquery.py, line 6 - F401 'ESQuerySet' imported but unused
* corehq/apps/es/tests/test_esquery.py, line 109 - E127 continuation line over-indented for visual indent
* corehq/apps/es/tests/test_esquery.py, line 134 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 135 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 146 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 147 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 148 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 151 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 10 - E201 whitespace after '['
* corehq/apps/es/tests/test_esqueryset.py, line 20 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 21 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 32 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 33 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 34 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 37 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 76 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 77 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 81 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 82 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 83 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/hqadmin/reporting/reports.py, line 160 - E302 expected 2 blank lines, found 1
* custom/opm/reports.py, line 390 - E127 continuation line over-indented for visual indent

**Comment by dimagimon** [comment]
There are 26 errors:

* corehq/apps/analytics/tasks.py, line 275 - E501 line too long (122 > 115 characters)
* corehq/apps/es/tests/test_aggregations.py, line 60 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 225 - E122 continuation line missing indentation or outdented
* corehq/apps/es/tests/test_aggregations.py, line 237 - W391 blank line at end of file
* corehq/apps/es/tests/test_esquery.py, line 6 - F401 'ESQuerySet' imported but unused
* corehq/apps/es/tests/test_esquery.py, line 109 - E127 continuation line over-indented for visual indent
* corehq/apps/es/tests/test_esquery.py, line 134 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 135 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 146 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 147 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 148 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 151 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 10 - E201 whitespace after '['
* corehq/apps/es/tests/test_esqueryset.py, line 20 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 21 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 32 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 33 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 34 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 37 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 76 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 77 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 81 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 82 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 83 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/hqadmin/reporting/reports.py, line 160 - E302 expected 2 blank lines, found 1
* custom/opm/reports.py, line 390 - E127 continuation line over-indented for visual indent

**Comment by dimagimon** [comment]
There are 23 errors:

* corehq/apps/es/tests/test_aggregations.py, line 60 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 225 - E122 continuation line missing indentation or outdented
* corehq/apps/es/tests/test_esquery.py, line 6 - F401 'ESQuerySet' imported but unused
* corehq/apps/es/tests/test_esquery.py, line 109 - E127 continuation line over-indented for visual indent
* corehq/apps/es/tests/test_esquery.py, line 134 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 135 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 146 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 147 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 148 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 151 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 10 - E201 whitespace after '['
* corehq/apps/es/tests/test_esqueryset.py, line 20 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 21 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 32 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 33 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 34 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 37 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 76 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 77 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 81 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 82 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 83 - E123 closing bracket does not match indentation of opening bracket's line
* custom/opm/reports.py, line 390 - E127 continuation line over-indented for visual indent

**Comment by dimagimon** [comment]
There are 23 errors:

* corehq/apps/es/tests/test_aggregations.py, line 60 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 225 - E122 continuation line missing indentation or outdented
* corehq/apps/es/tests/test_esquery.py, line 6 - F401 'ESQuerySet' imported but unused
* corehq/apps/es/tests/test_esquery.py, line 109 - E127 continuation line over-indented for visual indent
* corehq/apps/es/tests/test_esquery.py, line 134 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 135 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 146 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 147 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 148 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 151 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 10 - E201 whitespace after '['
* corehq/apps/es/tests/test_esqueryset.py, line 20 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 21 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 32 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 33 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 34 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 37 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 76 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 77 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 81 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 82 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 83 - E123 closing bracket does not match indentation of opening bracket's line
* custom/opm/reports.py, line 390 - E127 continuation line over-indented for visual indent

**Comment by dimagimon** [comment]
There are 23 errors:

* corehq/apps/es/tests/test_aggregations.py, line 60 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 225 - E122 continuation line missing indentation or outdented
* corehq/apps/es/tests/test_esquery.py, line 6 - F401 'ESQuerySet' imported but unused
* corehq/apps/es/tests/test_esquery.py, line 109 - E127 continuation line over-indented for visual indent
* corehq/apps/es/tests/test_esquery.py, line 134 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 135 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 146 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 147 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 148 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 151 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 10 - E201 whitespace after '['
* corehq/apps/es/tests/test_esqueryset.py, line 20 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 21 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 32 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 33 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 34 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 37 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 76 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 77 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 81 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 82 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 83 - E123 closing bracket does not match indentation of opening bracket's line
* custom/opm/reports.py, line 390 - E127 continuation line over-indented for visual indent

**Comment by dimagimon** [comment]
There are 24 errors:

* corehq/apps/es/aggregations.py, line 153 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 60 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 225 - E122 continuation line missing indentation or outdented
* corehq/apps/es/tests/test_esquery.py, line 6 - F401 'ESQuerySet' imported but unused
* corehq/apps/es/tests/test_esquery.py, line 109 - E127 continuation line over-indented for visual indent
* corehq/apps/es/tests/test_esquery.py, line 134 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 135 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 146 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 147 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 148 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 151 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 10 - E201 whitespace after '['
* corehq/apps/es/tests/test_esqueryset.py, line 20 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 21 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 32 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 33 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 34 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 37 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 76 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 77 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 81 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 82 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 83 - E123 closing bracket does not match indentation of opening bracket's line
* custom/opm/reports.py, line 390 - E127 continuation line over-indented for visual indent

**Comment by dimagimon** [comment]
There are 23 errors:

* corehq/apps/es/tests/test_aggregations.py, line 63 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 228 - E122 continuation line missing indentation or outdented
* corehq/apps/es/tests/test_esquery.py, line 6 - F401 'ESQuerySet' imported but unused
* corehq/apps/es/tests/test_esquery.py, line 109 - E127 continuation line over-indented for visual indent
* corehq/apps/es/tests/test_esquery.py, line 134 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 135 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 146 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 147 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 148 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 151 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 10 - E201 whitespace after '['
* corehq/apps/es/tests/test_esqueryset.py, line 20 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 21 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 32 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 33 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 34 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 37 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 76 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 77 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 81 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 82 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 83 - E123 closing bracket does not match indentation of opening bracket's line
* custom/opm/reports.py, line 390 - E127 continuation line over-indented for visual indent

**Comment by snopoke** [comment]
@esoergel @benrudolph

**Comment by snopoke** [comment]
buddy @biyeun

**Comment by dimagimon** [comment]
There are 23 errors:

* corehq/apps/es/tests/test_aggregations.py, line 63 - E131 continuation line unaligned for hanging indent
* corehq/apps/es/tests/test_aggregations.py, line 228 - E122 continuation line missing indentation or outdented
* corehq/apps/es/tests/test_esquery.py, line 6 - F401 'ESQuerySet' imported but unused
* corehq/apps/es/tests/test_esquery.py, line 109 - E127 continuation line over-indented for visual indent
* corehq/apps/es/tests/test_esquery.py, line 134 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 135 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 146 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 147 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 148 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esquery.py, line 151 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 10 - E201 whitespace after '['
* corehq/apps/es/tests/test_esqueryset.py, line 20 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 21 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 32 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 33 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 34 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 37 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 76 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 77 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 81 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 82 - E123 closing bracket does not match indentation of opening bracket's line
* corehq/apps/es/tests/test_esqueryset.py, line 83 - E123 closing bracket does not match indentation of opening bracket's line
* custom/opm/reports.py, line 390 - E127 continuation line over-indented for visual indent

File: corehq/apps/es/aggregations.py [hunk 1]
```diff
@@ -124,6 +129,9 @@ def __getattr__(self, attr):
         if sub_aggregation:
             return sub_aggregation.parse_result(self.result)
 
+    def __repr__(self):
+        return "Bucket(key='{}', doc_count='{})".format(self.key, self.doc_count)
```
Comment (esoergel) [inline corehq/apps/es/aggregations.py]: :+1:

File: corehq/apps/es/facets.py [hunk 2]
```diff
@@ -1,115 +0,0 @@
...
-    """
-    type = "date_histogram"
-
-    def __init__(self, name, datefield, interval, timezone=None):
-        self.name = name
-        self.params = {
-            "field": datefield,
-            "interval": interval,
-            "time_zone": timezone,
-        }
```
Comment (esoergel) [inline corehq/apps/es/facets.py]: :boom:

File: corehq/apps/es/es_query.py [hunk 3]
```diff
@@ -273,26 +268,25 @@ def fields(self, fields):
         """
             Restrict the fields returned from elasticsearch
 
-            Usage Note: As of ES 1.x, fields will only work on leaf nodes! It will no longer return an object,
-            e.g. field.*, to return an object refer to '.source'
+            Deprecated. Use `source` instead.
             """
-        query = deepcopy(self)
-        query._fields = fields
-        return query
```
Comment (esoergel) [inline corehq/apps/es/es_query.py]: Do you know if just making this method an alias for `source` would work?

**Comment by esoergel** [comment]
Looks great, thanks for doing this!

File: corehq/apps/es/es_query.py [hunk 4]
```diff
@@ -24,7 +24,7 @@
          .sort('received_on', desc=False)
          .size(self.pagination.count)
          .start(self.pagination.start)
-         .terms_facet('babies.count', 'babies_saved', size=10))
+         .terms_aggregation('babies.count', 'babies_saved'))
```
Comment (benrudolph) [inline corehq/apps/es/es_query.py]: This is great :baby: :angel:

File: corehq/apps/es/es_query.py (hunk 3 above)
Comment (snopoke) [inline corehq/apps/es/es_query.py]: Isn't that what I've done?

**Comment by czue** [comment]
monster stuff! :japanese_ogre:

**Pull request merged**